.linthtmlrc.json

```diff
@@ -1,3 +1,49 @@
 {
-  "extends": "@mate-academy/linthtml-config"
+  "attr-bans": [
```

Comment:
Pushing the **.linthtmlrc.json** file is generally not good. Because, the changes in this file  are generally only concerns you. 
<img width="952" alt="Adsız" src="https://user-images.githubusercontent.com/108900524/201060707-40bba4bb-6c52-42fb-baf9-69736acc49d2.png">
<img width="954" alt="Adsız2" src="https://user-images.githubusercontent.com/108900524/201060711-5bc33b2c-5b72-412c-a862-212d88baea78.png">